pkg/master/BUILD

```diff
@@ -156,6 +156,7 @@ go_test(
         "//vendor/k8s.io/client-go/rest:go_default_library",
         "//vendor/k8s.io/client-go/testing:go_default_library",
     ],
+    features = ["-race"],
```

Comment:
do we have issues tracking these racy tests?

Reply:
+1 if we don't, we definitely should

Reply:
Both are due to #39649.

Reply:
I wonder if gazelle should be automanaging the `-race` feature, given that it's a tag in the go source.

Reply:
I don't think so. It's a bad idea to use it in the way that we do.